src/services_models/reserves.jl

```diff
@@ -105,19 +105,17 @@ function add_constraints!(
     requirement = PSY.get_requirement(service)
     jump_model = get_jump_model(container)
     if parameters
-        container =
+        param_container =
             get_parameter(container, RequirementTimeSeriesParameter(), SR, service_name)
-        param = get_parameter_array(container)
+        param = get_parameter_column_refs(param_container, service_name)
```

Comment:
Among other things, something is broken here or in the way this container is initialized.